js/ui/date_box/ui.date_view.js

```diff
@@ -137,7 +137,7 @@ const DateView = Editor.inherit({
                 }
             });
         });
-        that._$rollersContainer.appendTo(that._wrapper());
+        that._$rollersContainer.appendTo(that._$wrapper);
```

Comment:
why no public method?

Reply:
that is 'date_view' own method. Actually i changed it accidentally and forget to return as it was:) but there is no difference

Reply:
let's retrieve it

Reply:
Done